src/pytest_bdd/steps.py

```diff
@@ -74,10 +74,10 @@ def get_step_fixture_name(step: Step) -> str:
 
 def given(
     name: str | StepParser,
-    converters: dict[str, Callable] | None = None,
+    converters: dict[str, Callable[[Any], Any]] | None = None,
```

Comment:
ditto

Reply:
I'm not sure whether `Callable[P, T]` is what is needed in this case. What would that imply for:

```python
@given(
    "Some step",
    converts={
        "id": int,
        "valid": bool,
        "list": lambda s: s.split(", "),
    },
)
```

Having a single `Callable[P, T]` would imply that they all have the same return type which is clearly incorrect.

Having said that, it might be correct to say that this should be refined to `Callable[[str], Any]`, unless you are aware of cases where something else than a string is passed to the convert?

Reply:
ah right, indeed it should be `Callable[[str], Any]`